src/Pagination.vue

```diff
@@ -43,21 +71,31 @@ export default {
 <style scoped>
 .VueCarousel-pagination {
   width: 100%;
-  float: left;
   text-align: center;
 }
 
 .VueCarousel-dot-container {
   display: inline-block;
   margin: 0 auto;
+  padding: 0;
 }
 
 .VueCarousel-dot {
-  float: left;
```

Comment:
I not understand why using float instead of `display: inline-block`

Reply:
Hmm, the original author must have preferred floats - but I agree I don' think we should be using them. Good find